shipcss.css

```diff
@@ -1,9 +1,62 @@
 :root {
-    --board-color: navy;
-    --border-color: lightskyblue;
+    /* --board-color: navy; */
+    --border-color: lightgray;
 }
+body {
+    background-image: url(../ships/photo/image\ 2.png);

```

Comment:
Please change path to `url('../photo/background.png');`
And filename respectively. Please don't use spaces in filenames and paths.

Reply:
solved